jvm-execution/src/main/scala/com/avast/server/toolkit/execution/LoggingUncaughtExceptionHandler.scala

```diff
@@ -0,0 +1,13 @@
+package com.avast.server.toolkit.execution
+
+import java.lang.Thread.UncaughtExceptionHandler
+
+import org.slf4j.LoggerFactory
+
+object LoggingUncaughtExceptionHandler extends UncaughtExceptionHandler {
+
+  private lazy val logger = LoggerFactory.getLogger("LoggingUncaughtExceptionHandler")
```

Comment:
I would prefer `getClass.getName` instead of the string constant

Reply:
It will add unnecessary `$` at the end of the logger name because it's a Scala `object`. I will keep this.